blocks/api/raw-handling/index.js

```diff
@@ -150,33 +187,21 @@ export default function rawHandler( { HTML, plainText = '', mode = 'AUTO', tagNa
 
 		doc.body.innerHTML = piece;
 
-		const transformsFrom = getBlockTransforms( 'from' );
-
-		const blocks = Array.from( doc.body.children ).map( ( node ) => {
-			const transformation = findTransform( transformsFrom, ( transform ) => (
-				transform.type === 'raw' &&
-				transform.isMatch( node )
-			) );
-
-			if ( transformation ) {
-				if ( transformation.transform ) {
-					return transformation.transform( node );
-				}
-
-				return createBlock(
-					transformation.blockName,
-					getBlockAttributes(
-						getBlockType( transformation.blockName ),
-						node.outerHTML
-					)
-				);
+		return Array.from( doc.body.children ).map( ( node ) => {
+			const { transform, blockName } =
+				find( rawTransformations, ( { isMatch } ) => isMatch( node ) );
```

Comment:
Does the destructure on the `find` result risk throwing an error when a match isn't found?

```
> var { foo, bar } = _.find( [ 1 ], _.stubFalse );
TypeError: Cannot destructure property `foo` of 'undefined' or 'null'.
    at repl:1:22
    at ContextifyScript.Script.runInContext (vm.js:59:29)
    at REPLServer.defaultEval (repl.js:242:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
    at Array.onLine (repl.js:468:10)
```

Reply:
See @mcsf's note above. :)